### PR TITLE
Implement `registry describe` and `dep describe`

### PR DESCRIPTION
**Pull request by hausdorff**

### CODE REVIEW

File: cmd/registry.go [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+	registryCmd.AddCommand(registryListCmd)
+	registryCmd.AddCommand(registryDescribeCmd)
+}
+
+var registryCmd = &cobra.Command{
+	Use:   "registry",
+	Short: `Manage registries for current project`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return fmt.Errorf("Command 'registry' requires a subcommand\n\n%s", cmd.UsageString())
+	},
```
Comment (jessicayuen) [inline cmd/registry.go]: Can you add a long description here about what registries are?

File: cmd/registry.go [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+		}
+
+		formatted, err := utils.PadRows(rows)
+		if err != nil {
+			return err
+		}
+		fmt.Print(formatted)
+		return nil
+	},
+	Long: ``,
```
Comment (jessicayuen) [inline cmd/registry.go]: Delete the `Long` and `Example` if not going to fill in. Same with Describe below.

File: cmd/registry.go [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+	Example: ``,
+}
+
+var registryDescribeCmd = &cobra.Command{
+	Use:   "describe",
+	Short: `Manage registries for current project`,
```
Comment (jessicayuen) [inline cmd/registry.go]: Shouldn't this be `Describe registries....`?

File: cmd/dep.go [hunk 4]
```diff
@@ -102,6 +106,65 @@ With these commands, a user can 'import "kspkg://nginx1"', and
 'import "kspkg://nginx2"' with no conflict.`,
 }
 
+var depDescribeCmd = &cobra.Command{
+	Use:   "describe [<registry-name/]<package-name>",
```
Comment (jessicayuen) [inline cmd/dep.go]: `/` should be `>` in `<registry-name/`

Comment (hausdorff) [reply cmd/dep.go]: I'm changing it to `[<registry-name>/]`, because the `/` is not optional if you include `registry-name`.

**Pull request merged**